src/cli/index.js

```diff
@@ -161,6 +161,14 @@ module.exports = function cli (params = {}, callback) {
               }
             }
           })
+
+          // Update scheduled events listeners
+          sandbox.scheduled.end()
```

Comment:
One bit about this implementation, if we're going to start adding on additional service operations, things might get wiggy bc async. I'm thinking maybe we should make this more deterministic and slap it all in a `run-series` call, wdyt?

All that said, this CLI module is long overdue for an overhaul (and testing), I'm open to ways to improve it!

Reply:
yeah i can add that